client/components/conditions/add-conditions.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useReducer } from 'react';
+import { Button } from '@ukhomeoffice/react-components';
+import Condition from './condition';
+import Field from '../field';
+import uuid from 'uuid/v4';
+import CONDITIONS from '../../constants/conditions';
+
+function CustomConditions({ conditions, onUpdate, onAdd, onRemove }) {
+  const handleOnUpdate = index => val => onUpdate(index, val);
+
+  return (
+    <>
```

Comment:
What's this new crazy?

Reply:
Can we stick to `<Fragment>` unless/until we're gonna replace the whole lot please?